### ISSUE TITLE
Overdraw in app

### ISSUE THREAD

**Issue by m-murad** [systers/PCSA-Android#371]
Overdraw causes performance and battery issues and must be removed.

@sandarumk 
Should I send a PR.

**Comment by sandarumk** [comment]
When you are reporting an issue, please be more specific. You should at least mention that where this happens and how you are going to fix it. There are several methods like removing unnecessary backgrounds in layouts and etc. to reduce overdraw.

**Comment by m-murad** [comment]
@sandarumk 
Actually there is some overdraw in almost every activity or fragment.
There are mainly two ways of reducing overdraw (in my knowledge). One is by removing unnecessary background. Second by not drawing overlapping views.

Since there are no overlapping views I will be reducing the overdraw by removing unnecessary background.

**Comment by sandarumk** [comment]
Regarding the overdraw reduction, Google developers have a nice article. 
For starters you can list out the activities and fragments where you can reduce overdraw.

**Comment by m-murad** [comment]
```
SignupActivity.java
SplashScreenActivity.java
UserSettingsActivity.java
Trustees.java
ContactPostStaff.java
PoliciesFragment.java
sexual_assault_awareness/MainFragment.java
SupportServicesFragment.java
```

These are some of the activities which have overdraw. There might be more.

**Comment by sandarumk** [comment]
Can you go through the code and find out all the activities and fragments which have overdraw?

**Comment by m-murad** [comment]
@sandarumk 
I have started working on the issue and will remove overdraw from all the activities.
I will also mention the activities that I change.

**Comment by sandarumk** [comment]
It is better to mention them here first.

**Comment by m-murad** [comment]
I have fixed the following activities -
1. `SplashScreenActivity.java`
2. `UserSettingsActivity.java`
3. `CircleIntroActivity.java`
4. `CircleOfTrustFragment.java`
5. `Trustees.java`
6. `ContactPostStaff.java`
7. `GlossaryFragment.java`
8. `PoliciesFragment.java`
9. `RadarFragment.java`
10. `SafetyPlanActivity.java`
11. `SafetyToolsFragment.java`
12. `sexual_assault_awareness/MainFragment.java`
13. `MythbustersFragment.java`
14. `SupportServicesFragment.java`
15. `GlossaryFragment.java`
16. `FAQArrayAdapter.java`
17. Fixed overdraw in Navigation Bar

@sandarumk 
Should I send a PR.

**Comment by free4murad** [comment]
@sandarumk 
Should I create a PR???

**Comment by sandarumk** [comment]
yeah go ahead

**Comment by sandarumk** [comment]
Fixed via #380
